ws-nextjs-app/pages/ws/languages.page.tsx

```diff
@@ -72,11 +75,20 @@ export const getServerSideProps: GetServerSideProps = async context => {
     props: {
       ...baseProps,
       pageType: HOME_PAGE,
-      service: 'pidgin',
-      pathname: '/pidgin',
+      service: 'ws',
+      pathname: '/ws/languages',
       status: data?.status,
       pageData: {
         ...data?.pageData,
+        metadata: {
+          ...data?.pageData?.metadata,
+          atiAnalytics: {
+            ...data?.pageData?.metadata?.atiAnalytics,
+            pageIdentifier: isLocalEnv
+              ? 'ws.languages.page'
+              : data?.pageData?.metadata?.atiAnalytics?.pageIdentifier,
```

Comment:
We need to do this for all environments - not just local. 

```suggestion
            pageIdentifier: 'ws.languages.page'
```

I think it's OK to do this here, rather than introducing custom logic to the BFF, right?